### PR TITLE
Epsilon / Conversant: Endpoint & supported openrtb version update

### PR DESCRIPTION
**Pull request by AaronColbyPrice**
Updating yaml descriptions for Epsilon/Conversant bidder to show updated support

### CODE REVIEW

**Comment by gargcreation1992** [comment]
@AaronColbyPrice Please resolve the merge conflicts.

File: static/bidder-info/epsilon.yaml [hunk 1]
```diff
@@ -13,6 +15,6 @@ capabilities:
       - video
 userSync:
   redirect:
-    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&rurl={{.RedirectURL}}"
+    url: "https://prebid-match.dotomi.com/match/bounce/current?version=1&networkId=72582&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&rurl={{.RedirectURL}}"
```
Comment (gargcreation1992) [inline static/bidder-info/epsilon.yaml:18]: works.

**Review by gargcreation1992 — DISMISSED** [review]

**Comment by gargcreation1992** [comment]
@AaronColbyPrice This branch has a conflict. Please resolve.

File: static/bidder-info/epsilon.yaml (hunk 1 above)
Comment (SyntaxNode) [inline static/bidder-info/epsilon.yaml:18]: Curious, why are you no longer checking for privacy policies?

Comment (AaronColbyPrice) [reply static/bidder-info/epsilon.yaml:18]: @SyntaxNode I'm not sure what you mean, what do you believe has changed in this pull request?

**Comment by AaronColbyPrice** [comment]
> @AaronColbyPrice This branch has a conflict. Please resolve.

All done

**Review by Sonali-More-Xandr — APPROVED** [review]

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**